packages/foreman/rubygem-rack/rubygem-rack.spec

```diff
@@ -73,8 +51,10 @@ cp -a .%{gem_dir}/* \
 mkdir -p %{buildroot}%{_bindir}
 cp -a .%{_bindir}/* \
         %{buildroot}%{_bindir}/
+
 find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 
+
```

Comment:
The double empty line is something we avoided in the previous placement of the `<% end -%>` tag.